PURL-SPECIFICATION.rst

```diff
@@ -206,44 +206,63 @@ The rules for each component are:
     in the canonical form
   - Each ``subpath`` segment MUST be a percent-encoded string
   - When percent-decoded, a segment:
+
     - MUST NOT contain a '/'
     - MUST NOT be any of '..' or '.'
     - MUST NOT be empty
+
   - The ``subpath`` MUST be interpreted as relative to the root of the package
 
 
 Character encoding
 ~~~~~~~~~~~~~~~~~~
 
-For clarity and simplicity a ``purl`` is always an ASCII string. To ensure that
+For clarity and simplicity, a ``purl`` is always an ASCII string. To ensure that
 there is no ambiguity when parsing a ``purl``, separator characters and non-ASCII
-characters must be UTF-encoded and then percent-encoded as defined at::
+characters MUST be UTF-encoded and then percent-encoded as defined at
+https://en.wikipedia.org/wiki/Percent-encoding and as further defined below.
+
+Use these rules for percent-encoding and decoding ``purl`` components.  Except
```

Comment:
Encoding is not specific to components. We are just defining how to encode and decode characters in a string. Sometimes we will refer to encoding for a whole component, some other times, we are stating the segments of a subpath are encoded individually.

Reply:
Good point -- changed to `percent-encoding and decoding the characters that comprise
a ``purl`` string` -- does that capture your thought @pombredanne ?